### PR TITLE
Optimize RowIDAllocator

### PR DESCRIPTION
**Pull request by humengyu2012**
## What is the purpose of the change
Fixed conflicts for creating RowIDAllocator.

## Brief change log
1. Set the default lock ttl to 1000ms and  remove random sleep time for flink 1.14 when creating RowIDAllocator.

## Verifying this change

Add some test for creating RowIDAllocator concurrently(50 threads).

## Does this pull request potentially affect one of the following parts:

- Dependencies (does it add or upgrade a dependency): no
- The public API: no
- The runtime per-record code paths (performance sensitive): no

## Documentation

- Does this pull request introduce a new feature? no
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: tidb/src/main/java/io/tidb/bigdata/tidb/allocator/RowIDAllocator.java [hunk 1]
```diff
@@ -133,24 +133,27 @@ private void set(@Nonnull List<BytePairWrapper> pairs, @Nonnull TiTimestamp time
...
-      // ignore
+      twoPhaseCommitter.commitPrimaryKey(
+          ConcreteBackOffer.newCustomBackOff(1000),
+          primaryPair.getKey(),
+          session.getTimestamp().getVersion());
+    } finally {
+      try {
+        twoPhaseCommitter.close();
+      } catch (Throwable ignored) {
+        // ignore
```
Comment (xuanyu66) [inline tidb/src/main/java/io/tidb/bigdata/tidb/allocator/RowIDAllocator.java:155]: better to print log

**Review by xuanyu66 — CHANGES_REQUESTED** [review]

File: tidb/src/main/java/io/tidb/bigdata/tidb/allocator/RowIDAllocator.java (hunk 1 above)
Comment (humengyu2012) [reply tidb/src/main/java/io/tidb/bigdata/tidb/allocator/RowIDAllocator.java:155]: It is copied from tispark, but it would be butter to print error log.

**Review by xuanyu66 — APPROVED** [review]

**Pull request merged**